src/Features/Core/Portable/Completion/CompletionService.ProviderManager.cs

```diff
@@ -53,15 +63,34 @@ public ProviderManager(CompletionService service)
                 return _lazyImportedProviders;
             }
 
-            public static ImmutableArray<CompletionProvider> GetProjectCompletionProviders(Project? project)
+            private ValueTask ProcessBatchAsync(ImmutableSegmentedList<IReadOnlyList<AnalyzerReference>> referencesList, CancellationToken cancellationToken)
```

Comment:
Can this cancellation token be used somewhere?

Reply:
hmm, it is used at [line 70 below](https://github.com/dotnet/roslyn/pull/64352/files/58c27a3d6ddad1dd7b2ef2dd529e2a8c46c94fd5#diff-b24e977d08d627b53afbd9cac5e1cdd8f940514c261b190705cff6ba7eb7a87aR70). Or maybe I misunderstood your comment?